--- a/cpp/CMakeLists.txt
+++ b/cpp/CMakeLists.txt
@@ -21,12 +21,12 @@ set (libphonenumber_VERSION_MAJOR 8)
 set (libphonenumber_VERSION_MINOR 12)
 set (libphonenumber_VERSION_PATCH 0)
 
-if (32BIT)
-  set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -m32")
-  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -m32")
-  set (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -m32")
-  set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -m32")
-endif ()
+#if (32BIT)
+#  set (CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -m32")
+#  set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -m32")
+#  set (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -m32")
+#  set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -m32")
+#endif ()
 
 # Helper functions dealing with finding libraries and programs this library
 # depends on.
@@ -466,14 +466,14 @@ if (${USE_RE2} STREQUAL "ON")
   list (APPEND LIBRARY_DEPS ${RE2_LIB})
 endif ()
 
-if (${USE_POSIX_THREAD} STREQUAL "ON" OR ((APPLE OR UNIX) AND ${USE_BOOST} STREQUAL "OFF" AND ${USE_STDMUTEX} STREQUAL "OFF"))
-  if(CMAKE_USE_PTHREADS_INIT)
-    list (APPEND CMAKE_C_FLAGS "-pthread")
-  endif()
-  if(CMAKE_THREAD_LIBS_INIT)
-    list (APPEND LIBRARY_DEPS ${CMAKE_THREAD_LIBS_INIT})
-  endif()
-endif ()
+#if (${USE_POSIX_THREAD} STREQUAL "ON" OR ((APPLE OR UNIX) AND ${USE_BOOST} STREQUAL "OFF" AND ${USE_STDMUTEX} STREQUAL "OFF"))
+#  if(CMAKE_USE_PTHREADS_INIT)
+#    list (APPEND CMAKE_C_FLAGS "-pthread")
+#  endif()
+#  if(CMAKE_THREAD_LIBS_INIT)
+#    list (APPEND LIBRARY_DEPS ${CMAKE_THREAD_LIBS_INIT})
+#  endif()
+#endif ()
 
 if (APPLE)
   list (APPEND COMMON_DEPS ${COREFOUNDATION_LIB} ${FOUNDATION_LIB})
